translations/ui-myprofile/en.json

```diff
@@ -12,6 +12,7 @@
   "settings.changePassword.enterValue": "Please enter a value.",
   "settings.changePassword.confirmPasswordMatchError": "New and confirm password does not match. Retype your password.",
   "settings.changePassword.wrongPassword": "Incorrect password. Retype your password.",
+  "settings.changePassword.lastTenPasswords": "Previously used password. Please enter a new password.",
```

Comment:
Please confirm with PO the exact text message. Also, make sure to add that string in other translation files. Use the English version in all of them.

Reply:
This text is listed in the task description.

Reply:
@VictorSoroka1 When `en.json` is updated, the new translation key and the English string are automatically added to Lokalise (the locale translation tool we're using).  The English string will be added to the non-`en.json` files automatically until there is a native language translation.

cc: @zburke : Thanks for pointing out this question to me.